announcementsAPI.json

```diff
@@ -363,12 +370,54 @@
         "properties": {
           "name": {
             "type": "string",
-            "nullable": true
+            "nullable": false
+          },
+          "sex":{
+            "$ref": "#/components/schemas/Sex"
           },
           "species": {
+            "type": "string",
+            "nullable": false
+          },
+          "breed": {
+            "type": "string",
+            "nullable": true
+          },
+          "birthday": {
+            "type": "string",
+            "format": "date-time"
+          },
+          "description": {
             "type": "string",
             "nullable": true
           },
+          "photo": {
```

Comment:
As discussed - photo cannot be posted with the same request as a pet, this is possible only via separate endpoint.

Reply:
I'll change this as soon as I merge other changes into this branch

Reply:
It's here, but it's an URI now.